src/cascade/input_data/configuration/construct_country.py

```diff
@@ -311,13 +309,12 @@ def get_measurement_data_by_sex(measurements):
         measurements (pandas.DataFrame): data for a specific measurement
 
     Returns:
-        dict: possible sex keys (-0.5, 0, 0.5) and measurement data as values
+        dict: possible sex keys (1, 2, 3, 4) and measurement data as values
     """
     measurements_by_sex = {}
 
-    for sex in (FEMALE, MALE, BOTH):
-
-        measurements_sex = measurements[measurements["x_sex"] == sex]
+    for sex in (FEMALE, MALE, BOTH, UNDEFINED):
```

Comment:
This is effectively a new feature.

Reply:
Yeah, I wasn't sure what to do. So I tried not to make a decision.

Reply:
It's a good choice. The modelers are clear that internally we should treat undefined as both.